kafka_exporter.go

```diff
@@ -250,15 +252,22 @@ func NewExporter(opts kafkaOpts, topicFilter string, groupFilter string) (*Expor
 	config.Metadata.RefreshFrequency = interval
 
 	client, err := sarama.NewClient(opts.uri, config)
-
 	if err != nil {
 		return nil, errors.Wrap(err, "Error Init Kafka Client")
 	}
 
 	glog.V(TRACE).Infoln("Done Init Clients")
+	admin, err := sarama.NewClusterAdmin(opts.uri, config)
```

Comment:
it's better to reuse the previous `client`

Reply:
And don't forget to close this `admin`

Reply:
If I understand you well you'd rather use an admin connection to the cluster for everything done by the exporter ?

Reply:
> And don't forget to close this `admin`
good catch